src/shared/local.env

```diff
@@ -0,0 +1,17 @@
+# kubectl apply -k config/crd/
```

Comment:
add some documentation about what this file is. Either here, or in the README.

Reply:
done